src/sa_web/static/js/routes.js

```diff
@@ -138,17 +150,40 @@ var Shareabouts = Shareabouts || {};
 
     // Open view for first step in multi-step form
     newPlace: function() {
+      console.log("newPlace");
+
       this.appView.newPlace();
     },
 
     viewLandmark: function(id) {
+      console.log("view landmark");
+
       this.appView.viewLandmark(id, { zoom: this.loading });
     },
 
+    // decide whether to route to the place view or the landmark view
+    // REFACTOR
+    // This method replaces viewLandmark and viewPlace
+    viewPlaceOrLandmark: function(datasetSlug, id, responseId) {
```

Comment:
Refactoring `viewLandmark` and `viewPlace` into a single method is not a priority yet, although it would be great for DRYness. But if it create too much complexity, don't worry about it. We can still implement our multiple datasets without the refactor.